yarn-project/end-to-end/src/e2e_inclusion_proofs_contract.test.ts

```diff
@@ -297,4 +292,8 @@ describe('e2e_inclusion_proofs_contract', () => {
   const getRandomBlockNumber = async () => {
     return deploymentBlockNumber + randomInt((await pxe.getBlockNumber()) - INITIAL_L2_BLOCK_NUM);
   };
+
+  const getRandomBlockNumberBeforeDeployment = () => {
+    return randomInt(deploymentBlockNumber - INITIAL_L2_BLOCK_NUM);
```

Comment:
```suggestion
    return randomInt(deploymentBlockNumber - INITIAL_L2_BLOCK_NUM) + INITIAL_L2_BLOCK_NUM;
```

Surprised this didn't result in test failure.

Reply:
Maybe I got lucky in the rng :stuck_out_tongue: